tox_docker/__init__.py

```diff
@@ -307,7 +308,7 @@ def tox_runtest_pre(venv):  # noqa: C901
                 healthcheck=healthcheck,
                 labels={"tox_docker_container_name": container_name},
                 links=links,
-                name=container_name,
+                name=container_name + f"{random.randint(0, 100000)}",
```

Comment:
Since you're using f-strings anyway, might be clearer to do
```suggestion
                name=f"{container_name}_{random.randint(0, 100000)}",
```

Reply:
To make the names both "more random" and predictable length (maybe this is just my OCD speaking), you could do:

```suggestion
                name=f"{container_name}_{random.randint(100000, 999999)}",
```

Reply:
Sure, I did check the tests quickly, but didn't find a right place to add new, I will check .

Reply:
I am using Zuul that runs jobs separately for each commit, and whenever I push bunch of commit, some will fail as they try to spin up container with same name, so, I have to retry the jobs sequentially, its PITA. And to implement retry I have to write a custom Zuul playbook (currently, uses built-in tox playbook).